sample/wallet/src/main/kotlin/com/walletconnect/sample/wallet/domain/NotificationHandler.kt

```diff
@@ -106,6 +104,9 @@ object NotificationHandler {
                     Triple(notification.messageId, notification.title, notification.body)
                 }
 
+                val url = (notifications.first() as? Notification.Decrypted)?.url
```

Comment:
Don't we want something like this instead where we want the first instance of a decrypted notification instead of if the first message is a decrypted notification?

```suggestion
val url = (notifications.first { it is Notification.Decrypted }?.url
```